### ISSUE TITLE
Modal: Resizable and moveable throughout screen

### ISSUE THREAD

**Issue by AdityaKhatri** [the-deep/deeper#935]

**Comment by eoglethorpe** [comment]
AC: 

*Be able to resize from any corner of a popup - not sure to what extent we should limit this. i.e can a user make it full screen. Mariam?
*Be able to move card when dragging and dropping. not sure if we should limit this to just a portion (i.e. top bar) or the whole popup. Mariam?

@AdityaKhatri @matthew-oglethorpe I feel that this is probably 2 cards? 1 for resize, 1 for move?

**Comment by AdityaKhatri** [comment]
@eoglethorpe , makes sense, but if we are doing it separately, resizing would be a bit weird as we would need to figure out which point to resize it by (center, top-right, top-left, ...). If we do them together, we can let the user do whatever they want. :1234:

**Comment by eoglethorpe** [comment]
@AdityaKhatri sure, i was just trying to be agile ;)

@Mariam-Asatryan seems i can tag you now

**Comment by mariam-yu** [comment]
@Ewan-Oglethorpe @AdityaKhatri not sure if it's me misunderstanding Zube, but what's the context? Which cards are we talking about?

General rule though, limit what the users can do, even if they think they want full flexibility. i.e., if we can resize it for them , let's do that, but anyway, can you guys share some screenshots? Thanks!

**Comment by AdityaKhatri** [comment]
Hi @Mariam-Asatryan , we are talking about in general all modals through out DEEP, but I do agree on not providing flexibility to users.
We haven't started this task yet.
This was a feature request by Patrice, so maybe we setup a call with him to know more why he wants this feature and if there is a workaround?
@eoglethorpe , what do you think?

**Comment by mariam-yu** [comment]
@AdityaKhatri sounds good to discuss with Patrice. If it's in general for modals, probably the best rule would be to make them as standard as possible, one size that goes around whatever content it shows with minimal interactions and buttons. But let me know more.

**Comment by mariam-yu** [comment]
Final discussion was: make them movable but not resizable. Hope works for everyone, and we can explore further in more detail.

**Comment by AdityaKhatri** [comment]
Should we provide the moveable feature to all modals in DEEP or should it be selective?

**Comment by mariam-yu** [comment]
@AdityaKhatri we still will tackle the comments. I would say if it's a separate module in dev don't touch it yet. For everything else, consistency is good. Do you guys have a list by chance of all the modals/popups?

**Comment by AdityaKhatri** [comment]
@Mariam-Asatryan , there are about ~71 unique Modal, and ~70 confirmation Modals from a quick analysis.

**Comment by mariam-yu** [comment]
Woa! @AdityaKhatri What's a unique modal?

**Comment by eoglethorpe** [comment]
mo' modals mo' problems

**Comment by AdityaKhatri** [comment]
@Mariam-Asatryan , confirmation modal are pretty similar to each other.
By unique modal, I mean modals that are not similar to each other and have different purposes.
